sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormTrainingClient/FormTrainingClientLiveTests.cs

```diff
@@ -127,6 +228,37 @@ public async Task StartTrainingFailsWithInvalidPrefix()
             Assert.AreEqual("2014", ex.ErrorCode);
         }
 
+        [Test]
+        public async Task StartTrainingWithLabelsDisplayName()
```

Comment:
Should we add a tests for the error conditions, where the user supplies a non existing model Id, or a "failed" status model Id for creating a composed model?

Reply:
Test added